src/Symfony/Component/Translation/Tests/DependencyInjection/TranslationPassTest.php

```diff
@@ -25,26 +25,33 @@ public function testValidCollector()
         $loader = (new Definition())
```

Comment:
(marking the legacy test as `@legacy` with `@expectedDeprecation` for the default `$readerServiceId` value that will change in 4.0.